cmd/wallet-server/startcmd/start.go

```diff
@@ -522,10 +528,24 @@ func getKeyServerParams(cmd *cobra.Command) (*keyServerParameters, error) {
 		return nil, fmt.Errorf("ops key server url : %w", err)
 	}
 
+	var useRemoteKMS bool
+
+	useRemoteKMSStr, err := cmdutils.GetUserSetVarFromString(
+		cmd, useRemoteKMSFlagName, useRemoteKMSEnvKey, true)
+	if err != nil {
+		return nil, fmt.Errorf("use remote kms : %w", err)
+	}
+
+	switch useRemoteKMSStr {
+	case "Y", "y", "yes", "YES", "true", "TRUE":
```

Comment:
Simplify this to just `true` or `false` and use `strconv.ParseBool()`

Reply:
updated